client/task.go

```diff
@@ -241,10 +242,13 @@ func (t *task) Pid() uint32 {
 }
 
 func (t *task) Start(ctx context.Context) error {
-	ctx, span := tracing.StartSpan(ctx, "task.Start",
-		tracing.WithAttribute("task.id", t.ID()),
-	)
+	ctx, span := tracing.StartSpan(ctx, tracing.Name("client.task", "Start"),
```

Comment:
I am thinking it belongs to refactoring and it could break exsisting span name. let's do it in the follow-up. thanks